assets/css/src/global.css

```diff
@@ -128,7 +128,17 @@ i {
 }
 
 blockquote {
+	border-left: 4px solid #000;
```

Comment:
Thinking these colors should be variables to be consistent.

Reply:
In this case, we replicate the style from _Quote_ Blick defined in a external stylesheet. Using a variable will not make our styles consistent a unless we overload the border color also for the _Quote_ block. I think add a variable and overload the border left color for the  _Quote_ block.